### DISCUSSION TITLE
Run SplineGNN

### DISCUSSION THREAD

**Discussion by liyuntong9** [pyg-team/pytorch_geometric#2757] [Q&A]
Are there hyperparameters with excellent performance available to run `SplineConv` model on `Citeseer` dataset?
When I tune the hyperparameters, I obtained the best test accuracy of 77.1%. 
However, I learned that the test accuracy can reach 79.2% from the website [paperwithcode](https://www.paperswithcode.com/sota/node-classification-on-citeseer) .
I follow this dataset splitting setting: 500 nodes are randomly selected for validation, 1000 for test, and the remaining nodes are used for training.
Please help me with some advice.

**Comment by rusty1s** [comment]
We didn't evaluate `SplineCNN` on `CiteSeer` in our paper, so I'm not sure who submitted this result. As such, I don't know which hyperparameters to use for the given test accuracy :(

**Reply by liyuntong9** [reply]
What a pity! Thanks for your reply.
